types/react-plotly.js/index.d.ts

```diff
@@ -1,27 +1,23 @@
 // Type definitions for react-plotly.js 2.2
 // Project: https://github.com/plotly/react-plotly.js#readme
 // Definitions by: Jon Freedman <https://github.com/jonfreedman>
+//                 Mitchell Grice <https://github.com/gricey432>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.8
+// TypeScript Version: 3.4
```

Comment:
I think you can leave this as 2.8 - this should be the minimum version of TypeScript that is compatible with these type definitions.

Reply:
Ah yep, my misunderstanding there